README.md

```diff
@@ -8,5 +8,22 @@ planner : MaxlPlanner (devloped at Mowito)
 Keyboard innput can be given at run time<br/> 
 robot will update path taking obstacle avoidance in account as goal is changed by keys i , l j to move it forword backword rigth left 
 
+# How to launch
+### System requirement
+Ubuntu 18/20<br/>
+ROS Melodic/Neotic (Ros1)<br/>
+Rviz (present by default if not [refer](http://wiki.ros.org/rviz/UserGuide))<br/>
```

Comment:
....refer to the link (add URL to the word link)

Do the same for rest of the places as well